cmd/install/assets/hypershift_operator.go

```diff
@@ -62,6 +62,9 @@ var (
 	// privileged is used to set the container security
 	// context to run container as unprivileged.
 	privileged = false
+
+	// readOnlyVerbs are RBAC related verbs limited to read actions
```

Comment:
Not sure that this change is necessary, you could reduce your diff by leaving it out. It's not like we will be changing the list at any time (and therefore benefiting from having a slice for it).